### PR TITLE
feat(preset-website): provide new "website" preset - INNO-633

### PR DESCRIPTION
**Pull request by yhuard**
"website" preset is the new main preset

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit a8ab3abd1fee06f207e6fff250451e22b68595c5

https://deploy-preview-639--europa-component-library.netlify.com

**Review by kalinchernev — DISMISSED** [review]
It's good for me :)

File: src/systems/ec/ec-preset/ec-preset-webtools/ecl-builder.config.js [hunk 1]
```diff
@@ -1,28 +1,18 @@
...
-const outputFolder = isProd ? './dist' : './build';
+const outputFolder = path.resolve(__dirname, isProd ? './dist' : './build');
 
 const nodeModules = path.resolve(__dirname, '../../../../../node_modules');
 
 // SCSS includePaths
 const includePaths = [nodeModules];
 
 module.exports = {
-  scripts: [
```
Comment (kalinchernev) [inline src/systems/ec/ec-preset/ec-preset-webtools/ecl-builder.config.js]: :+1:

**Review by kalinchernev — APPROVED** [review]
Thanks!

**Pull request merged**